imagetest/test_suites/disk/disk_resize_test.go

```diff
@@ -23,6 +23,7 @@ const (
 
 // TestDiskResize Validate the filesystem is resized on reboot after a disk resize.
 func TestDiskResize(t *testing.T) {
+	utils.LinuxOnly(t)
```

Comment:
I think this should be supported on windows too  https://cloud.google.com/compute/docs/disks/resize-persistent-disk

Reply:
Ditto on todo

Reply:
it is probably supported, however the test currently assumes linux with the markerfile and a unix call in verifyDiskSize().

I think windows disk resizing would make the most sense in a follow up PR: I've added "TODO" in the comments

Reply:
Not sure what's up with the test setup, but I get this failure running the test on windows:
```
<testcase classname="disk-windows-2016" name="TestDiskResize" time="0">
			<failure type="Failure">FileIOError: stat /disk64.exe: no such file or directory</failure>
</testcase>
```
I don't think it's skipping correctly

Reply:
it was because of the build directive on the first two lines, and because the unix import fails the build on windows. Both should be fixed with the latest commit